contracts/test/RFQ.t.sol

```diff
@@ -513,6 +570,18 @@ contract RFQTest is StrategySharedSetup {
         bytes memory makerSig,
         bytes memory userSig
     ) internal view returns (bytes memory payload) {
-        return abi.encodeWithSelector(rfq.fill.selector, order, makerSig, userSig);
+        bytes4 selector = rfq.fill.selector;
```

Comment:
Suggest to have another params for `_genFillPayload()` or can split into another payload getting function to be more specific instead of reading from the global `isMakerOnlyApproveRFQ` flag.